pkg/cmd/cli/describe/describer.go

```diff
@@ -618,7 +619,13 @@ const policyRuleHeadings = "Verbs\tResources\tResource Names\tExtension"
 func describePolicyRule(out *tabwriter.Writer, rule authorizationapi.PolicyRule, indent string) {
 	extensionString := ""
 	if rule.AttributeRestrictions != (runtime.EmbeddedObject{}) {
-		extensionString = fmt.Sprintf("%v", rule.AttributeRestrictions)
+		extensionString = fmt.Sprintf("%#v", rule.AttributeRestrictions.Object)
+
+		buffer := new(bytes.Buffer)
+		printer := NewHumanReadablePrinter(true)
+		if err := printer.PrintObj(rule.AttributeRestrictions.Object, buffer); err == nil {
+			extensionString = strings.TrimSpace(buffer.String())
```

Comment:
Passing `true` to the printer eliminates the extra column headings.  You usually won't that in a describe.